public/src/components/channelManagement/epicTests/epicTestEditor.tsx

```diff
@@ -233,22 +238,24 @@ const EpicTestEditor: React.FC<EpicTestEditorProps> = ({
         </div>
       )}
 
-      {epicEditorConfig.allowCustomVariantSplit && test.variants.length > 1 && (
-        <div className={classes.sectionContainer}>
-          <Typography variant={'h3'} className={classes.sectionHeader}>
-            Variants split
-          </Typography>
-          <div>
-            <TestVariantsSplitEditor
-              variants={test.variants}
-              controlProportionSettings={test.controlProportionSettings}
-              onControlProportionSettingsChange={onControlProportionSettingsChange}
-              onValidationChange={onVariantsSplitSettingsValidationChanged}
-              isDisabled={!editMode}
-            />
+      {epicEditorConfig.allowCustomVariantSplit &&
+        hasControl(test.variants) &&
+        test.variants.length >= 2 && (
```

Comment:
Just updated this logic as we shouldn't show the variant split editor if there isn't a control

Reply:
this logic is repeated on line 111 and 243, should we have a `hasNonControl` function as well?